aikau/src/main/resources/alfresco/editors/TinyMCE.js

```diff
@@ -390,7 +390,8 @@ define(["dojo/_base/declare",
          }
          if (this._resizeWhenInitialized)
          {
-            this.onResize();
+            // See AKU-775 - adding a short timeout here allows everything to complete initialization before resizing
+            setTimeout(lang.hitch(this, this.onResize), 100);
```

Comment:
We're still just guessing that it'll be ready ... perhaps something a little more concrete? There's a more specific solution at http://stackoverflow.com/questions/6287350/how-to-tell-if-tinymce-has-been-initated but even if not that, we could have a while loop that checks every few hundred milliseconds whether it's been loaded yet before continuing?